src/components/base.js

```diff
@@ -30,6 +30,20 @@ class BaseComponent extends Component {
     this.logger = LoggerFactory.getLogger(`UI ${this.name}`);
     this.logger.debug(`Initialized`);
   }
+
+  /**
+   * Gets config param value
+   * @param {string} property property name
+   * @returns {string} config value
+   */
+  getConfig(property: string): ?string {
+    try {
+      return this.config.components[this.name][property];
```

Comment:
isn't it better to solve the config object on init then to try-catch on each access?
IMO - better init the config in the constructor once with validation that it exist and then have the getConfig just return the property
```javascript
   constructor(obj?: Object = { config: {} }) {
     ...
     this.config = (obj.config.components && obj.config.components[this.name]) || {};
      ...
    }
```
```javascript
getConfig(property: string): ?string {
   return this.config[property];
}
```

this way a component will also carry only its config and not all components config